### PR TITLE
Paginated Help Embed

### PR DESCRIPTION
**Pull request by YoussefExplicit**
- Used for of when reacting
- Made pages a function to stop lots of repition

- Need to work on using 1 reactioncollector. Probably going to try that, and in the function, check for the reaction.

### CODE REVIEW

File: commands/system/help.js [hunk 1]
```diff
@@ -20,6 +20,7 @@ class Help extends Command {
     let page = 1;
     let n = 0;
     let num = 0;
+    let reactor;
```
Comment (ghost) [inline commands/system/help.js]: You define reactor but never use it anywhere ?

**Comment by ghost** [comment]
Running it gives a error of :- 

```
{ RangeError [EMBED_FIELD_VALUE]: MessageEmbed field values may not exceed 1024 characters or be empty.
    at MessageEmbed.addField (/Users/****/Documents/Discord.js/Misaki-Help/node_modules/discord.js/src/structures/MessageEmbed.js:180:54)
    at Help.run (/Users/****/Documents/Discord.js/Misaki-Help/commands/general/help.js:42:10)
    at module.exports.run (/Users/****/Documents/Discord.js/Misaki-Help/events/message.js:71:9)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:160:7) [Symbol(code)]: 'EMBED_FIELD_VALUE' }
```

**Review by ghost — COMMENTED** [review]
Shift the pages function to inside the class and call it  via this.pages(.....)

File: commands/system/help.js [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+      await message.channel.send('I don"t have permission to remove reactions, please do this manually.');
+    }
+
+    if(msg2.embeds[0].title) {
+      if (msg2.embeds[0].title.startsWith('Page')) {
+        for (const emoji of emojis) await msg2.react(emoji);
+      }
+    }
+
+    const nextpage = msg2.createReactionCollector(
```
Comment (kyranet) [inline commands/system/help.js:90]: You're still using a lot of reaction collectors.

**Comment by Stitch07** [comment]
I'm not a fan of so many reaction collectors.   Should be possible with just 1.

**Review by kyranet — CHANGES_REQUESTED** [review]
Everything from #64 really. You're still using a lot of collectors.

File: commands/system/help.js (hunk 2 above)
Comment (YoussefExplicit) [reply commands/system/help.js:90]: That’s solved. Need to push a new update when I get home

**Pull request closed without merging**